docs/dev/adding_instruments.rst

```diff
@@ -149,6 +149,60 @@ Power supplies
 PSUs typically can measure the *actual* current and voltage, as well as have settings for the voltage level and the current limit.
 To keep naming clear and avoid confusion, implement the properties :code:`current`, :code:`voltage`, :code:`voltage_setpoint` and :code:`current_limit`, respectively.
 
+Managing status codes or other indicator values
+***********************************************
+Often, an instrument features one or more collections of specific values that signal some status, an instrument mode or a number of possible configuration values.
+Typically, these are collected in mappings of some sort, as you want to provide a clear and understandable value to the user, while abstracting away the raw data, think :code:`ACQUISITION_MODE` instead of :code:`0x04`.
+The mappings normally are kept at module level (i.e. not defined within the instrument class), so that they are available when using the property factories.
```

Comment:
Can't you access them at class level (e.g. AwG401.SomeEnum instead of self.SomeEnum)?
Because for the setter a class enum is better.

Or (for a `control`), the class may contain the module level enum.

Anyway, we can adjust the documentation according to upcoming experience.

Reply:
In my experience this does not work reliably (also for lambdas, etc), but we can sure test in a more systematic way. I propose to do that another time. Changing the docs to an improved approach should always be welcome. :-)

Reply:
See #736